lib/datadog/tracing/contrib/dalli/instrumentation.rb

```diff
@@ -37,6 +37,9 @@ def request(op, *args)
 
                 span.set_tag(Tracing::Metadata::Ext::NET::TAG_TARGET_HOST, hostname)
                 span.set_tag(Tracing::Metadata::Ext::NET::TAG_TARGET_PORT, port)
+
+                span.set_tag(Datadog::Tracing::Metadata::Ext::DB::TAG_SYSTEM, 'memcached')
```

Comment:
Because these database tag values (e.g. `'memcached'`) are well-defined and effectively enumerables, I suggest we move them to named constants.

I think `contrib/ext.rb` would be a good place for them.